platform/src/main/java/org/stellar/anchor/platform/config/PropertySep10Config.java

```diff
@@ -117,6 +117,30 @@ void validateConfig(Errors errors) {
           "sep10-jwt-timeout-invalid",
           "The sep10.jwt_timeout must be greater than 0");
     }
+
+    byte[] nonce = new byte[64];
+
+    try {
+      new ManageDataOperation.Builder(String.format("%s %s", homeDomain, "auth"), nonce).build();
+    } catch (IllegalArgumentException iaex) {
+      errors.rejectValue(
```

Comment:
Can we have a special error for when domains are too long? Current error message is hard to debug.

Reply:
Hey Gleb, this is still a draft.

Reply:
I will modify the error messages.

Reply:
Ah sorry about that, I thought it wasn't a draft. Sounds good, thank you

Reply:
No problem. 💯